Applications/WebPortal/WebApp/library/source/pages/dashboard_page/dashboard_controller.tsx

```diff
@@ -94,9 +95,8 @@ export class DashboardController extends React.Component<Properties, State> {
 
   private renderAccountPage = () => {
     const model = (() => {
-      const pattern = Path.pathToRegexp(
-        '/account/:id(\\d+)?', [], { end: false });
-      const match = pattern.exec(window.location.pathname);
+      const match = DashboardController.ACCOUNT_PATTERN
```

Comment:
Break the line up on the function call:

    const match = DashboardController.ACCOUNT_PATTERN.exec(
      ...);